main/src/com/google/refine/operations/column/ColumnAdditionByFetchingURLsOperation.java

```diff
@@ -140,6 +150,7 @@ public void write(JSONWriter writer, Properties options)
         writer.key("onError"); writer.value(TextTransformOperation.onErrorToString(_onError));
         writer.key("delay"); writer.value(_delay);
         writer.key("cacheResponses"); writer.value(_cacheResponses);
```

Comment:
Not sure if this should under the model level? maybe should a preference?

Reply:
Not sure I understand (sorry). Do you mean the headers should be set at a preference level? I did consider this, but each request might need different headers, and if you haven't recorded the headers used when making a specific request you can't undo/redo in a predictable way

Reply:
@ostephens but we could have PER PROJECT Preferences as well as GLOBAL Preferences.
@jackyq2015 Didn't your model/metadata re-work actually afford that now for us ? or would there be more work needed ?

Reply:
At the moment these are per operation. I'm not clear that having these at a project or global level is that useful? Even within a single project different operations might require different headers? What's the demand for having these per project or global?

That said I don't have any particular objection, but it would need a new issue